ccpp/data/GFS_typedefs.F90

```diff
@@ -6415,6 +6425,7 @@ subroutine diag_create (Diag, IM, Model)
     allocate (Diag%snohfa  (IM))
     allocate (Diag%transa  (IM))
     allocate (Diag%sbsnoa  (IM))
+    allocate (Diag%paha    (IM))
```

Comment:
According to https://github.com/NCAR/ccpp-physics/pull/729, some of the additional variables are only used when NoahMP is active. You may want to conditionally allocate these when lsm==lsm_noahmp to save memory.